sktime/transformations/base.py

```diff
@@ -644,10 +645,11 @@ def update(self, X, y=None, Z=None, update_params=True):
 
         # 3. internal only has Series but X is Panel: loop over instances
         elif X_input_scitype == "Panel" and "Panel" not in X_inner_scitypes:
-            if y is not None:
+            if y is not None and self.get_tag("y_inner_mtype") != "None":
                 raise ValueError(
-                    "no default behaviour if _fit does not support Panel, "
-                    " but X is Panel and y is not None"
+                    "no default behaviour if _fit does not support Panel and "
+                    'self.get_tag("y_inner_mtype") is not "None",'
```

Comment:
If this is intended as a message to the user it might not be as helpful as anticipated. Consider adding a suggestion about what to change.